setup.py

```diff
@@ -33,7 +33,7 @@
     name="google-auth-oauthlib",
     version=version,
     author="Google Cloud Platform",
-    author_email="jonwayne+google-auth@google.com",
+    author_email="googleapis-auth@google.com",
```

Comment:
```suggestion
    author_email="googleapis-packages@google.com",
```

I added suggestion to set `author_email` to the email used in other `googleapis` packages which receives a lot of spam. 

Feel free to keep the one that you proposed but note that spam emails may increase.

Reply:
Yes committed your suggestion, thank you!